projects/Mallard/src/authentication/lib/Authorizer.ts

```diff
@@ -110,6 +111,7 @@ class Authorizer<
                 },
             })
         } catch (e) {
+            crashlyticsService.captureException(e)
```

Comment:
could you put this line inside the  `errorService.captureException`function to reduce duplication?

Reply:
We can do that, but my thought was to keep them separate now with a view that either `crashlytics` or `errorService` will be removed in the future.

Reply:
the errorService does some helpful stuff with capturing consent which could be useful to keep even after deprecating Sentry?

Reply:
discussed in catchup: we don't need the GDPR stuff as this is checked before crashlytics is initialised. The `errorService` doesn't do any other useful stuff